js/damage.js

```diff
@@ -120,6 +120,9 @@ function getDamageResult(attacker, defender, move, field) {
     if (typeEffectiveness === 0) {
         return {"damage":[0], "description":buildDescription(description)};
     }
+    if (move.name === "Sky Drop" && (defender.weight >= 200 || (defender.type1 === "Flying" || defender.type2 === "Flying"))) {
```

Comment:
I forgot this file is also used for Gen 5 damage calcs, so this should actually be something like `(move.name === "Sky Drop" && (defender.type1 === "Flying" || defender.type2 === "Flying" || (gen >= 6 && defender.weight >= 200)))`.